doc/designs/xof.md

```diff
@@ -78,13 +78,17 @@ EVP_DigestSqueeze(ctx, out, outlen).
 ##### Pros
 
   - Seems like a better name.
+  - The existing function does not change, so it is not affected by logic that needs to
+    run for the multi squeeze case.
+  - The behaviour of the existing API is the same.
+  - At least one other toolkit uses this approach.
 
 ##### Cons
 
   - Adds an extra API.
   - The interaction between the 2 API's needs to be clearly documented.
-     - A call to EVP_DigestSqueeze() after EVP_DigestFinalXOF() would fail.
-     - A call to EVP_DigestFinalXOF() after the EVP_DigestSqueeze() would fail? Is this confusing.
+  - A call to EVP_DigestSqueeze() after EVP_DigestFinalXOF() would fail.
+  - A call to EVP_DigestFinalXOF() after the EVP_DigestSqueeze() would fail? Is this confusing.
```

Comment:
This actually needs to fail as this would be second attempt to finalize and that is an error.